projects/Mox/test.nix

```
@@ -0,0 +1,61 @@
+{
+  sources,
+  pkgs,
+  ...
+}:
+{
+  name = "mox";
+
+  nodes = {
+    machine =
+      { ... }:
+      rec {
+        imports = [
+          sources.modules.ngipkgs
+          # Sources module path depends on your folder structure
+          ./module.nix
```

We can just import the module and the example:

```suggestion
          sources.modules.ngipkgs
          sources.modules.services.mox
          sources.examples.Mox.mox
```